templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -432,6 +435,11 @@ export default {
     isMultiContentNodeChild() {
       return this.parent && this.parent.mediaType == "multi-content"
     },
+    isPageMultiConentNodeChild() {
+      return (
+        !!this.isMultiContentNodeChild && this?.parent?.presentationStyle === "page"
```

Comment:
`this` should never be undefined so that question mark is not needed
```suggestion
        !!this.isMultiContentNodeChild && this.parent?.presentationStyle === "page"
```